lib/groebner.gd

```diff
@@ -485,6 +485,8 @@ DeclareOperation("GroebnerBasis",
 DeclareOperation("GroebnerBasis",[IsPolynomialRingIdeal,IsMonomialOrdering]);
 DeclareGlobalFunction("GroebnerBasisNC");
 
+DeclareSynonym("GrobnerBasis",GroebnerBasis);
```

Comment:
`lib/helpbase.gi` defines global variable `TRANSATL` which is currently 
```
            [ [ "atalogue", "atalog" ],
              [ "olour", "olor" ],
              [ "entre", "enter" ],
              [ "isation", "ization" ],
              [ "ise", "ize" ],
              [ "abeling", "abelling" ],
              [ "olvable", "oluble" ],
              [ "yse", "yze" ] ] );
```
Do we want to add ["Groebner","Grobner"] pair there so the search `??Grobner` will return all entries with Groebner? The only thing that stops me is that I do not know whether this spelling (or misspelling) is relevant to the Atlantic ocean...